tsdb/engine/b1/b1.go

```diff
@@ -772,10 +776,10 @@ func marshalWALEntry(key []byte, timestamp int64, data []byte) []byte {
 
 // unmarshalWALEntry decodes a WAL entry into it's separate parts.
 // Returned byte slices point to the original slice.
-func unmarshalWALEntry(v []byte) (key []byte, timestamp int64, data []byte) {
+func unmarshalWALEntry(v []byte) (key, timestamp, data []byte) {
```

Comment:
Ah... I see now.  Was confused how `b.Put` was compiling when it requires a `[]byte`